.github/workflows/ci.yml

```diff
@@ -124,7 +129,7 @@ jobs:
       - uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: "17"
+          java-version: ${{ matrix.java }}
```

Comment:
Some jobs in the matrix don't have a version set up at all, we can do 17 for them